bitvm/src/chunk/api_runtime_utils.rs

```diff
@@ -552,8 +548,12 @@ pub(crate) fn execute_script_from_signature(
             for index in index_of_bitcommitted_msg {
                 let sig_data = &bitcom_sig_arr[index as usize];
                 let sig_preimage = match sig_data {
-                    SigData::SigHash(signature) => signature.to_compact_script(),
-                    SigData::Sig256(signature) => signature.to_compact_script(),
+                    SigData::SigHash(signature) => Wots16::compact_signature_to_raw_witness(
+                        &Wots16::signature_to_compact_signature(signature),
+                    ),
+                    SigData::Sig256(signature) => Wots32::compact_signature_to_raw_witness(
+                        &Wots32::signature_to_compact_signature(signature),
```

Comment:
Why do we have the mappping:

- `SigHash` -> `Wots16`
- `Sig256` -> `Wots32`

Wouldn't it better to have all of them `{Sig,Wots}{16,32}`?